test/modules/corrado/bigint_inversion_error.chpl

```diff
@@ -0,0 +1,35 @@
+use BigInteger;
```

Comment:
Totally reasonable to not know this already, but when we add tests of standard and package modules, we tend to put them in `test/library/[standard | packages]/<name of the module>`.  So in this case, we'd want to add this test to `test/library/standard/BigInteger`.  That directory already has a `skipif` on it, so you wouldn't need to add one for this test specifically :)